### ISSUE TITLE
Stencil breaks with optifine in the Nether

### ISSUE THREAD

**Issue by TenableRelic90** [ferriarnus/VivecraftForge#178]
I have a few other mods, nothing that changes things visually or could cause this glitch to happen (I don't think anyways). I can't find any other sources on this glitch so I'm coming here to report it on my own. Playing Vivecraft on 1.20.1, going into the nether causes a strange glitch that puts some sort of glitched object in my face. It glitches around or stays still depending on what shader is on. But is very seizure inducing and cannot be fixed or worked around. The second I go in the nether, it starts, the second I leave, so does it. It changes colors depending on shader as well. It sits directly on the face as if a combination of random shapes are on my character model constantly glitching in and out of view. I can temporarily get rid of it when in the shader slection options in the optifine menu. But the second I leave the settings hud again, it comes back instantly. I can't prevent it by having shaders off, its happening in multiple different worlds including online hosted servers, and I cannot find a fix or way to prevent it.

**Comment by fayer3** [comment]
as stated on discord, I would like a video of the issue, a latest.log and a mod list

**Comment by fayer3** [comment]
closing as no response

**Comment by MrDed9979** [comment]
this glitch happened to me, but im using optifine but that does nothing. maybe load up version 14 and go into the nether. i used  steam VR runtime. there was no crashing, so no logs

**Comment by fayer3** [comment]
can you post a video and log?
the latest log is always generated.minecraft/logs/latest.log

**Comment by MrDed9979** [comment]
let me look

**Comment by MrDed9979** [comment]
and also i just got off vr and i aint going back on 💀

**Comment by MrDed9979** [comment]
maybe tmrw

**Comment by fayer3** [comment]
that is fine

**Comment by MrDed9979** [comment]
maybe we should talk in discord about it when i manage to get discord back
friend me in advance: mrded9978

**Comment by jpv776** [comment]
Forge 1.21.1 and this still occurs, this seems to be an issue with the FOV Eye Stencil under Stereo Rendering. Disabling it seems to fix this but at the cost of a performance hit due to rendering more pixels.

**Comment by fayer3** [comment]
yeah, I narrowed it down to optifine breaking the triangle render mode when entering the nether
